spec/README.md

```diff
@@ -0,0 +1,6 @@
+![image](https://user-images.githubusercontent.com/298359/171659507-8dd4c49d-f6bd-43de-b0c1-e5ab2d83bd65.png)
```

Comment:
can we use this instead just in case?
![Untitled design(22)](https://user-images.githubusercontent.com/22844059/171661556-ba455f74-c1d5-4d3c-890a-b4bfba97f26e.png)



https://user-images.githubusercontent.com/22844059/171661393-a234931c-b8b1-4958-8186-f0936e432128.png

Reply:
done